po/nl.po

```diff
@@ -10309,14 +10296,12 @@ msgid "Name of the report to run\n"
 msgstr "Naam van het rapport om te genereren.\n"
 
 #: gnucash/gnucash-cli.cpp:115
-#, fuzzy
 msgid "Specify export type\n"
-msgstr "2. Het type import selecteren"
+msgstr "2. Het type import selecteren\n"
```

Comment:
No "2. " in source

Reply:
Oops, I fixed the newline in that string, but didn't verify the rest of its contents. It's export also rather than import. Should be fixed now.